src/Slide/Slide.jsx

```diff
@@ -15,8 +15,10 @@ const Slide = class Slide extends React.PureComponent {
     index: PropTypes.number.isRequired,
     innerClassName: PropTypes.string,
     innerTag: PropTypes.string,
-    naturalSlideHeight: PropTypes.number.isRequired,
-    naturalSlideWidth: PropTypes.number.isRequired,
+    /* eslint-disable react/require-default-props */
```

Comment:
Just a question,  is there a way to accomplish this without ignoring the rule?

Reply:
Not sure to be honest. I am pretty sure I tried, but basically `CarouselPropTypes.slideSize` can be either optional or required and eslint struggles with that.
But the main takeaway is that:
* If it's required, then it doesn't need default props
* If it's optional, then default doesn't need to be anything as it can be undefined (it doesn't matter)